packages/integration-tests/src/test-e2e/deployment.test.ts

```diff
@@ -118,19 +119,26 @@ void describe('deployment tests', { concurrency: testConcurrencyLevel }, () => {
         });
 
         after(async () => {
-          await testProject.tearDown(sandboxBackendIdentifier);
+          // await testProject.tearDown(sandboxBackendIdentifier);
```

Comment:
left over from testing?

Reply:
Whoops yes, did that to have time to verify shared secret naming when running locally.